kernel/src/main/scala/cats/kernel/std/Util.scala

```diff
@@ -85,4 +86,5 @@ object StaticMethods {
     }
     true
   }
+  // scalastyle:on magic.number
```

Comment:
I might be reading or understanding this incorrectly, but it looks like the `scalastyle:off` has `return`, but `scalastyle:on` has `magic.number`. Is this a mistake?